fe/fe-core/src/main/java/org/apache/doris/mtmv/MTMVTaskManager.java

```diff
@@ -58,10 +58,11 @@ public class MTMVTaskManager {
 
     private static final Logger LOG = LogManager.getLogger(MTMVTaskManager.class);
 
-    // jobId -> pending tasks, one job can dispatch many tasks
-    private final Map<Long, PriorityBlockingQueue<MTMVTaskExecutor>> pendingTaskMap = Maps.newConcurrentMap();
+    // mvId -> jobId -> pending tasks, one job can dispatch many tasks
+    private final Map<Long, Map<Long, PriorityBlockingQueue<MTMVTaskExecutor>>> pendingTaskMap = Maps
+            .newConcurrentMap();
 
-    // jobId -> running tasks, only one task will be running for one job.
+    // mvId -> running tasks, only one task will be running for one job.
```

Comment:
can we here use `mvId -> jobId -> taskid`?
One mv will contains many jobs but only one in running status.
One job will contain many tasks and tasks can be executed paralism